### PR TITLE
Hw 5 hover leave events

### PR DESCRIPTION
**Pull request by aberezhk**
Added solution for HW 5 on top of changes in previous pull request:

New button on buttons page that changes it style on mouse enter and gets back to original state on mouse leave. On click alert is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
     <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;700&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="css/main.css">
     <link rel="icon" href="img/redi.png">
 </head>
 <body>
 <div class="headerBar">
     <ul class="headerBar__links">
-        <li><a class="headerBar__link headerBar__link--active" href="/">Home</a></li>
+        <li><a class="headerBar__link headerBar__link--active" href="./">Home</a></li>
```
Comment (BTBTravis) [inline index.html:14]: not sure this will work in git pages but lets see

File: buttons/index.html [hunk 2]
```diff
@@ -4,36 +4,44 @@
...
+        <div class="btns__innner">
+            <p>How are you?</p>
+            <button class="btn" id="awesomeButton">Awesome 🤩</button>
+            <button class="btn" id="goodButton">Good 👍</button>
+            <button class="btn" id="badButton">Bad 👎</button>
+            <p id="answer">❓</p>
         </div>
     </div>
+    <div class="break"></div>
+    <button class="btn" id="strangeButton">Strange Button</button>
```
Comment (BTBTravis) [inline buttons/index.html:38]: Screenshot 2020-04-19 at 22 07 33

**Review by BTBTravis — APPROVED** [review]
Nice work! See you tomorrow for class.

File: buttons/buttons.css [hunk 3]
```diff
@@ -20,3 +25,40 @@
     padding: 0.5em 1em;
     font-size: 20px;
 }
+
+.burger-btn {
```
Comment (MrTim) [inline buttons/buttons.css:29]: oh this burger would make me smile all the time now ))))))

**Review by MrTim — APPROVED** [review]
:+1:

File: buttons/buttons.css (hunk 3 above)
Comment (aberezhk) [reply buttons/buttons.css:29]: Actually I saw 2 more same type of burger buttons (1 was even made with a separate jpg of burger).

**Pull request merged**